command/server/config.go

```diff
@@ -68,6 +68,12 @@ type Config struct {
 
 	DisableSentinelTrace    bool        `hcl:"-"`
 	DisableSentinelTraceRaw interface{} `hcl:"disable_sentinel_trace"`
+
+	HostnameHeader    bool        `hcl:"-"`
+	HostnameHeaderRaw interface{} `hcl:"hostname_header"`
```

Comment:
When it is a config that is indicating Vault "to do something", we might want to go with using verbs in the names to make them self explanatory. In that sense, we may want to do something like `add_hostname_response_header` and `add_raft_node_id_response_header`.

This would be inline with `disable_mlock`, `disable_sentinel_trace`.

Reply:
Or actually `enable_response_header_response_header` and `enable_response_header_raft_node_id` to set the prefix for more response headers we may want to add in future.

Reply:
Sure, makes sense. I'll update it.